index.html

```diff
@@ -117,13 +117,15 @@ <h1>Contact me</h1>
                     project
                     that
                     needs coding. I’d love to help with it </p>
-                <form action="https://formspree.io/f/xoqbdrjg" method="post" class="form-container form-padding">
+                <form id="form" action="https://formspree.io/f/xoqbdrjg" method="post" class="form-container form-padding">
                     <input type="text" name="fullname" maxlength="30" placeholder="Enter your fullname" required />
-                    <input type="email" name="email" placeholder="Enter your email" required />
+                    <input type="email"  name="email" placeholder="Enter your email" required />
+                    <span class="email-invalid"></span>
                     <textarea placeholder="Write your message here" name="message" maxlength="500" required></textarea>
                     <div class="submit-btn">
-                        <button class="btn">Get In Touch</button>
+                        <button class="btn" type="submit">Get In Touch</button>
                     </div>
+                    <span class="error-message"> Error </span>
```

Comment:
- [ ] Following the project requirements, the error message should show up close to the "submit" button.

Reply:
![image](https://user-images.githubusercontent.com/50754458/195336086-ed944397-e10a-4255-a5a2-0e0cf101cfed.png)
Obviously, the error message is showing up close to the submit button. Did you try that? This is how it looks like.